docs/install/command-line.md

```diff
@@ -89,43 +89,69 @@ The IPFS team manages the [dist.ipfs.tech website](https://dist.ipfs.tech/) to h
 
 ### macOS
 
-:::tip M1-based Macs
-You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead of the `amd64` binary listed in these instructions.
-:::
+1. Decide which macOS binary to download from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo) by determining whether your system uses Apple or Intel CPUs. On most MacOS systems, you can determine the system specs by doing the following:
```

Comment:
> ...by determining whether your system uses Apple or Intel CPUs.

This can be changed to "...by determining whether your system uses an Apple or Intel CPU." Just has better flow in my opinion.

- MacOS should always have a lower case "m"; i.e, "macOS"
- In my opinion the backticks around the link should be removed, but I see that format was used elsewhere in this doc so we'll keep it for now.

Reply:
This all makes sense, changed in https://github.com/ipfs/ipfs-docs/pull/1289/commits/8662022f8929748d6fd0d4f1001d07eb3fb08c55